main.config.js

```diff
@@ -400,34 +366,106 @@ export const team = [
-        position: 'Full-stack js Astronaut',
-        description: '',
+        position: 'HTML Coder',
+        description: 'All the tasks related to the development of web interfaces lie upon my shoulders. Friendly team and flexible working hours are what motivates me to keep going as a part of the KeenEthics team.',
         avatar: 'max_s.jpg',
       },
-      {
-        name: 'Vadym S.',
-        position: 'Full-stack js Astronaut',
-        description: '',
-        avatar: 'vadym_s.jpg',
-      },
       {
         name: 'Dmytro P.',
-        position: 'Full-stack js Astronaut',
-        description: '',
+        position: 'Full-Stack Software Developer',
+        description: 'My central objective is to achieve the optimal result. At KeenEthics, I find the personal and professional freedom to be the most valuable benefit, which inspires and motivates me to devote myself to the common goal.',
         avatar: 'dmytro_p.jpg',
       },
+      {
+        name: 'Anton M.',
+        position: 'Full-Stack Software Developer',
+        description: 'my job is to provide maintainable clean code for the projects I am assigned to and to make sure that it is timely tested. The KeenEthics team is always willing to change for the sake of our common good and for the benefit of our partners. Fresh fruits in our kitchen are also a huge plus!',
+        avatar: 'astronauts.jpg',
+      },
+      {
+        name: 'Iryna K.',
+        position: 'Lead Generation Manager',
+        description: 'I am responsible for lead generation and other sales activities, including communication with leads, marketing activities such as creating marketing items (brochures, email campaigns etc.), as well as social media marketing and content management on LinkedIn and Twitter. I like that the KeenEthics team members are always open for a small chat during the coffee break and ready to give a professional help.',
+        avatar: 'astronauts.jpg',
+      },
+      {
+        name: 'Marian Y.',
+        position: 'Quality Assurance Engineer',
+        description: 'I test software. The greatest and my most favorite thing about working at KeenEthics is the professionalism of our team. The atmosphere in the office is utterly favorable for creative thinking, brainstorming, learning, and advancing one’s professional expertise.',
+        avatar: 'astronauts.jpg',
+      },
+      {
+        name: 'Ilona-Mariia Sh.',
+        position: 'Full-Stack Software Developer',
+        description: 'Communication with clients and marketing activities are my major job objectives. I am responsible for communicating with the broad audience through Instagram, and I am glad to show people how fun and cozy the atmosphere in our team is. KeenEthics inspires me with an excellent atmosphere, great relationships between people, flexibility, and the ability to develop myself personally and professionally.',
+        avatar: 'astronauts.jpg',
+      },
+      {
+        name: 'Max F.',
```

Comment:
@BorysKoblents , it seems Max F. is duplicated - he is present [above](https://github.com/keenethics/keenethics/pull/112/files#diff-18bcbd48ee0718098b53d37bdecddbc6R301)